.github/workflows/ci.yml

```diff
@@ -111,7 +111,7 @@ jobs:
       - name: build
         run: |
           for /f "usebackq delims=" %%i in (`vswhere.exe -latest -property installationPath`) do if exist "%%i\Common7\Tools\vsdevcmd.bat" call "%%i\Common7\Tools\vsdevcmd.bat" -arch=x64 -host_arch=x64
-          win_build_vs.bat ..\b /64 /CI /S /A /PDH /SPLUNK /GCP /ELASTIC /K /L /R /Z /N /RO /PR /PYTHON_SCRIPTING /LUA_SCRIPTING /MQTT /SCCACHE /NINJA
+          win_build_vs.bat ..\b /64 /CI /PDH /R /N /RO /SCCACHE /NINJA
```

Comment:
is this intentional?

Reply:
Also it seems that elastic and gcp extension defaults were missing so I added them in 26d8bafcda3458c8166e269dd7ccacaee506f142